src/containers/Token/DEXPairs/index.js

```diff
@@ -138,6 +138,13 @@ const DEXPairs = props => {
       <div className="title"> {t('top_trading_pairs')}</div>
       <div className="pairs-table">
         <table>
+          <thead>
+            <tr>
+              <th className="pair-header">Pair</th>
+              <th>Issuer</th>
+              <th className="stats-header">Stats</th>
```

Comment:
Is there a better header we can use than "Stats"? Seems a bit vague to me

Reply:
I'm open to suggestions. The component shows the lowest and highest offers currently available. In the future we might add the average exchange rate as a tooltip.

Reply:
What about something like "Offer Range"?

Reply:
Sure that works for me.

Reply:
Updated in latest